frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationBody.tsx

```diff
@@ -43,9 +43,11 @@ export const UtilizationBody: React.FC<UtilizationBodyProps> = ({ timestamps, ch
   );
 
   return (
-    <div className="co-utilization-card__body pf-l-grid">
-      {axis}
-      {children}
+    <div className="co-dashboard-card__body--no-padding">
```

Comment:
Are we adding a wrapping div here just to remove padding? Could we just handle this in the css, do we need an extra div?

Reply:
Im trying to stay consistent with all other cards which are always using structure as following:
```
<DashboardCard>
  <DashboardCardHeader>
  ...
  </DashboardCardHeader>
  <DashboardCardBody>
  ...
  </DashboardCardBody>
</DashboardCard>
```